### ISSUE TITLE
Creating lists failes

### ISSUE THREAD

**Issue by knut-hildebrandt** [akirk/enable-mastodon-apps#9]
Tried to create lists with through Fedilab but always failed with an error message that said that an error occurred.

**Comment by akirk** [comment]
Indeed, lists are not yet supported but I have started work a while ago on https://github.com/akirk/friends-roles which will lay the basis for this.
